misc_docs/syntax/extension_embed_raw_expression.mdx

```diff
@@ -0,0 +1,34 @@
+---
+id: "embed-raw-expression"
```

Comment:
Why is it called `embed-*`? Wouldn't be `raw-expression` be enough?

I guess it's because you are embedding raw JS code... i am not sure if the extra term helps in the understanding though.

Reply:
Thanks for the feedback, I was undecided with this one, but I think your point about it not helping with understanding makes sense.

I've changed to the following:

`%raw` == "JavaScript expression" extension point
`%%raw` == "Top level JavaScript" extension point

Also, since the syntax uses the term `raw`. Do you think should be included in the name?

`%raw` == "Raw JavaScript expression" extension point
`%%raw` == "Raw top level JavaScript" extension point

I was thinking that "JavaScript" and "raw" here are almost synonymous in this context.

Happy to refine further.

Reply:
^^^ I think this is all resolved with your suggestions.